### DISCUSSION TITLE
My organization has been suspended

### DISCUSSION THREAD

**Discussion by 0x99sk** [GitbookIO/community#747] [0. General bugs and questions] [answered]
closed

**Comment by 0x99sk** [comment]


**Reply by addisonschultz** [reply]
We'll look into this! Apologies in the meantime. Will get support right on this.

**Reply by john-gitbook** [reply]
Hey @gid-MjAyNDA4MDE, Could you try again? It should work now.

**Reply by john-gitbook** [reply]
Glad you can sign in again. 
Your account was flagged by our system due to a few factors, including the use of multiple devices and proxies. These security measures are necessary to help prevent abuse. To reduce the chances of being flagged in the future, we recommend minimizing the number of devices connected to your account and limiting the use of different IP addresses.

**Reply by luigi3141** [reply]
> Glad you can sign in again. Your account was flagged by our system due to a few factors, including the use of multiple devices and proxies. These security measures are necessary to help prevent abuse. To reduce the chances of being flagged in the future, we recommend minimizing the number of devices connected to your account and limiting the use of different IP addresses.

We are experiencing this now on our site https://docs.rethinkable.xyz/

We had a new team member login from a different device/location and now none of us are able to log in and our site says "The organization has been suspended". We contacted support but are still waiting to hear back. 

Are you able to escalate the recovery for us?

**Reply by addisonschultz** [reply, marked as answer]
Hi @luigi3141 - it should be back up now. Apologies for the inconvenience!

**Reply by serkor1** [reply]
@addisonschultz I have the exact same issue as @luigi3141 and @0x99sk - I can login to Gitbook with another account, but not my own. I am (also) using a VPN. The site was: https://slmetrics.gitbook.io/api-docs

**Reply by aikoii** [reply]
@serkor1 👋,  just replied to your email. You should have access now!
